docs/dictionary-and-concepts/dictionary-metering.md

```diff
@@ -12,43 +12,61 @@ The terms listed here are all terms relating to the topics of defining metering.
 | Accounting Point |  A domain under balance responsibility where Energy Supplier change can take place and for which commercial business processes are defined. **Additional information:** This is a type of Metering Point. |
 | Asset Type | The technology used on a specific metering point  |
 | **B** |   |
+| Business Reason Code | Indicates the business context, e.g. E23 = 'Periodic metering' used with time series |
 | **C** |   |
+| Created Date Time | A date and time property used in all messages exchanged with Green Energy Hub. It indicates when a particular message was created |
```

Comment:
Who sets this? Isn't this set by the actor sending the message in, as it is a property in the XML? In that case I don't think this is something we should ever use, but instead set a "receival time" once a message is received.

Reply:
It is a property in both ebIX xml and CIM xml. It is set by the sender of the message, so this goes for both inbound and outbound time series message. Inbound set by the market participant, outbound set by Green Energy Hub. I will add some extra text to the definition :)

Reply:
Exactly, so we need to be very careful actually using this property in the inbound message, as it can be anything.

Reply:
Agreed. I don't expect the value to be used for anything once the time series is handed over to async processing. So for now it is just a term in the dictionary :) 
Ps. We set RequestDate as the "receival time" you mentioned.